applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -28,6 +28,7 @@
 from adjoint_vms_sensitivity_2d import AdjointVMSSensitivity2D
 from hdf5_io_test import HDF5IOTest
 from test_statistics_process import IntegrationPointStatisticsTest
+from test_mass_conservation_check_process import TestMassConservationUtility
```

Comment:
In the `FluidDynamicsApplication` we tend to put `_test` at the end of file name (this helps to search the files by alphabetical order). Would you mind changing it to the previous naming convention?

Reply:
done